lib/modules/datasource/sbt-package/index.ts

```diff
@@ -143,15 +143,31 @@ export class SbtPackageDatasource extends Datasource {
     return result;
   }
 
-  async getReleases({
-    packageName,
-    registryUrl,
-  }: GetReleasesConfig): Promise<ReleaseResult | null> {
+  private isGitlab(url: string): boolean {
+    const parsedUrl = parseUrl(url);
+    if (parsedUrl) {
+      const { hostname } = parsedUrl;
+      if (hostname === 'gitlab.com') {
```

Comment:
this only works for the official gitlab.com but not for self hosted versions.

Reply:
@zharinov Shouldn't we use our `detectPlatform` function instead?

https://github.com/renovatebot/renovate/blob/dab6a9c908f1dd31a4fa0580939ba41f79d546fc/lib/modules/platform/util.ts#L14

that would allow supporting private gitlab instances too